jax/errors.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2021 Google LLC
```

Comment:
Can you follow the `_src` pattern? This module should be a trivial forwarding module.

(This saves having to move this code later.)

Reply:
Done